static/js/public/formValidation.js

```diff
@@ -0,0 +1,32 @@
+function formValidation() {
+  const form = document.querySelector(".marketo-form");
+  if (form) {
+    initFormValidation(form);
+  }
+}
+
+function initFormValidation(form) {
+  var startTime = Date.now();
+  var delta = null;
+
+  var timer = setInterval(function() {
```

Comment:
I'd move this below the `spamCheck` definition block (line 24) to ensure the element exists by the time the timer starts.

Can also be a `const`.